tests/bench.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Maybe we could add a commentary here explained the purpose of this file (ie the fact that it is to be ran by ocaml-bench).